pkgs/os-specific/linux/cpupower/default.nix

```diff
@@ -1,22 +1,21 @@
-{ stdenv, fetchurl, kernel, coreutils, pciutils, gettext }:
+{ stdenv, buildPackages, fetchurl, kernel, pciutils, gettext }:
 
 stdenv.mkDerivation {
   name = "cpupower-${kernel.version}";
 
   src = kernel.src;
 
-  buildInputs = [ coreutils pciutils gettext ];
+  nativeBuildInputs = [ gettext ];
+  buildInputs = [ pciutils ];
 
   configurePhase = ''
     cd tools/power/cpupower
-    sed -i 's,/bin/true,${coreutils}/bin/true,' Makefile
-    sed -i 's,/bin/pwd,${coreutils}/bin/pwd,' Makefile
-    sed -i 's,/usr/bin/install,${coreutils}/bin/install,' Makefile
+    sed -i 's,/bin/true,${buildPackages.coreutils}/bin/true,' Makefile
+    sed -i 's,/bin/pwd,${buildPackages.coreutils}/bin/pwd,' Makefile
+    sed -i 's,/usr/bin/install,${buildPackages.coreutils}/bin/install,' Makefile
   '';
 
-  buildPhase = ''
-    make
-  '';
+  makeFlags = [ "CROSS=${stdenv.cc.targetPrefix}" ];
```

Comment:
This shouldn't be `stdenv.hostPlatform.config`, right? (The practical difference is the trailing `-`, the conceptual difference is what I wrote identifies the platform, whereas that is just the disambiguating prefix of the compiler and need not be anything in particular.

Reply:
The value of `CROSS` is used as the compiler and linker prefix in the Makefile (see [here](/torvalds/linux/blob/master/tools/power/cpupower/Makefile#L101)), so isn't `targetPrefix` the right value for that?

Reply:
Yeah sounds good